client/components/deposits-overview/index.tsx

```diff
@@ -88,8 +89,12 @@ const DepositsOverview: React.FC = () => {
 		);
 	}
 
-	// This card isn't shown if there are no deposits, so we can bail early.
-	if ( ! isLoading && deposits.length === 0 ) {
+	if (
+		! hasCompletedWaitingPeriod &&
+		availableFunds === 0 &&
+		pendingFunds === 0
+	) {
+		// If the account has not received any transactions before completing the waiting period, don't show the deposits overview section.
```

Comment:
This comment might be easier to read above the if (less indentation). 

Also … getting my head around it, I think it means…

- Don't show deposits card if
  - The site is still in the waiting period
  - AND 
  - The site has no transactions

Or 

```
If still in new account waiting period and account has no transactions,
don't render deposits card (nothing to show).
```

Reply:
👍 nice suggestion, resolved in 6e9e181ebfbbe95c940ec2cee7c8186bdd8b9709